src/applications/vaos/reducers/newAppointment.js

```diff
@@ -412,12 +413,20 @@ export default function formReducer(state = initialState, action) {
         flowType: FLOW_TYPES.REQUEST,
         reasonRemainingChar: REASON_MAX_CHARS.request,
       };
-    case FORM_CLINIC_PAGE_OPENED: {
+    case FORM_FETCH_FACILITY_DETAILS:
       return {
         ...state,
         loadingFacilityDetails: true,
       };
-    }
+    case FORM_FETCH_FACILITY_DETAILS_SUCCEEDED:
+      return {
+        ...state,
+        loadingFacilityDetails: false,
```

Comment:
Can we make this `facilityDetailsStatus` or something and use the FETCH_STATUS statuses? We'll also need to handle the failure case for this action.